### PR TITLE
Merge common-header into apps

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Include a summary of the change. If this is fixing a defect, ensure to link to the issue this is fixing. 

## Motivation and Context
Why is this change required? What problem does it solve?

## How Has This Been Tested?
Describe in detail how you tested your changes. Include details of your testing environment and link(s) for reviewers to validate.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
   <!-- app css -->
   <!-- build:css css/ea.min.css -->
-  <link rel="stylesheet" type="text/css" href="bower_components/common-header/dist/css/rise.min.css">
+  <link rel="stylesheet" type="text/css" href="tmp/css/rise.min.css">
```
Comment (ezequielc) [inline web/index.html]: @alex-deaconu Curious, do we need to add css to tmp/ folder? The css-build task adds to both tmp/ and dist/ but I am wondering if we need both or we could rely on the dist/ only. Thanks

File: web/scripts/common-header/config/config.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (ezequielc) [inline web/scripts/common-header/config/config.js]: @alex-deaconu For now it is fine, but at some point we should probably move this to the regular apps config/. Thanks

File: web/index.html (hunk 1 above)
Comment (alex-deaconu) [reply web/index.html]: @ezequielc `web/index.html` cannot access the `dist/` folder when accessing it in the local server. That's why I opted for the extra folder.

Let me know if you have any suggestions on how to work around that.

Thanks.

File: web/scripts/common-header/config/config.js (hunk 2 above)
Comment (alex-deaconu) [reply web/scripts/common-header/config/config.js]: @ezequielc agreed, I just added the prod config for simplicity. Thanks.

Comment (alex-deaconu) [reply web/scripts/common-header/config/config.js]: Just a note, this config is needed for the CH distribution build. If we moved everything to Apps config, we would need to update that as well.

File: web/index.html (hunk 1 above)
Comment (ezequielc) [reply web/index.html]: @alex-deaconu Oh, good point. I don't have better suggestions. I am good with it. Thanks!

**Comment by alex-deaconu** [comment]
Moving to https://github.com/Rise-Vision/rise-vision-apps/pull/1279

**Pull request closed without merging**